components/automate-cli/cmd/chef-automate/certRotate.go

```diff
@@ -169,12 +181,25 @@ func certRotatePG(publicCert, privateCert, rootCA string, infra *AutomteHAInfraD
 	remoteService := "postgresql"
 
 	// Creating and patching the required configurations.
-	config := fmt.Sprintf(POSTGRES_CONFIG, privateCert, publicCert, rootCA)
+	var config string
+	if nodeFlag.node != "" {
+		if rootCA != "" {
```

Comment:
since we are ignoring the CA for a given node case, it is not mandatory for providing the CA in the command. Please alter the mandatory check in getCerts() method

Reply:
updated